cmd/mdatagen/internal/metadata.go

```diff
@@ -310,6 +311,20 @@ type Attribute struct {
 	Optional bool `mapstructure:"optional"`
 }
 
+func (a *Attribute) Unmarshal(parser *confmap.Conf) error {
+	err := parser.Unmarshal(a)
+	if err != nil {
+		return err
+	}
+
+	// default enabled to true
+	if !parser.IsSet("enabled") {
```

Comment:
looks like this is also applied to the resource attributes. The resource attributes with missing `enabled` fields now get enabled=true. That's why tests are failing. 

I think we should make the `enabled` required and fill all missing spots in contrib to unblock this PR

Reply:
While looking into this, I found that telemetry attributes will be also affected. We expose all attributes in the user config including those that are applied to internal telemetry, but they don't have any aggregation capabilities at this moment... This is another blocker. Let me think about